sycl/test-e2e/Matrix/joint_matrix_hip_half_fill.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %{build} -fsycl -fsycl-targets=amd_gpu_gfx90a -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 %s -o %t.out
+// RUN: %{run} %t.out
+
+// REQUIRES: gpu-amd-gfx90a
+// REQUIRES: aspect-fp16
+
+#include "joint_matrix_hip_fill.hpp"
```

Comment:
I think you meant to include joint_matrix_hip_apply.hpp here

Also, would it be possible to e.g rename the functions e.g.:

`hip_matrix_mfma` -> `hip_matrix_fill` in fill header
`hip_matrix_mfma` -> `hip_matrix_apply` in apply header

And then also simply have on joint_matrix_hip_gfx90a.cpp test file that calls these functions?